### DISCUSSION TITLE
Remove Heading

### DISCUSSION THREAD

**Discussion by Sangarang** [community/community#23460] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/7e643b1be75d01150b5ba953ecee03d3a3346b44.png" data-download-href="/uploads/short-url/i26U6JwQpzMKDK9jYyY310GRGok.png?dl=1" title="E165E1FD-F6A2-4A13-958A-5B89118625CA.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/36474922/181077210-53e7d206-cc59-4d07-a9b6-bc05298a5dbe.png" alt="E165E1FD-F6A2-4A13-958A-5B89118625CA.png" data-base62-sha1="i26U6JwQpzMKDK9jYyY310GRGok" width="690" height="297" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/7e643b1be75d01150b5ba953ecee03d3a3346b44_2_690x297.png, https://aws1.discourse-cdn.com/github/original/1X/7e643b1be75d01150b5ba953ecee03d3a3346b44.png 1.5x, https://aws1.discourse-cdn.com/github/original/1X/7e643b1be75d01150b5ba953ecee03d3a3346b44.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/7e643b1be75d01150b5ba953ecee03d3a3346b44_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">E165E1FD-F6A2-4A13-958A-5B89118625CA.png</span><span class="informations">1000×431 2.08 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>What is the heading above the ‘hello’ and how do I remove it?
<p>I have got no idea what it is and it’s really annoying. The word ‘test’ above is the repository name by the way.</p>

**Comment by birdcar** [comment, marked as answer]
<p>The header you’re seeing there is because GitHub Pages uses the Primer theme by default to publish your Pages site:</p>
<p><a href="https://github.com/pages-themes/primer" rel="nofollow noopener">https://github.com/pages-themes/primer</a></p>
<p>Primer will include your repository’s name in an &lt;h1&gt; tag if your page’s title is different from your site’s title (or in this case, your page has no title):</p>
<p><a href="https://github.com/pages-themes/primer/blob/master/_layouts/default.html#L12-L14" rel="nofollow noopener">https://github.com/pages-themes/primer/blob/master/_layouts/default.html#L12-L14</a></p>
<p>If you’d like the remove that header, you’ll want to <a href="https://help.github.com/articles/customizing-css-and-html-in-your-jekyll-theme/#customizing-your-jekyll-themes-html-layout" rel="nofollow noopener">customize your Jekyll theme’s HTML</a>. In order to do that you’ll need to copy this file into your repository inside a folder called _layouts:</p>
<p><a href="https://github.com/pages-themes/primer/blob/master/_layouts/default.html" rel="nofollow noopener">https://github.com/pages-themes/primer/blob/master/_layouts/default.html</a></p>
<p>After you’ve done that, you can remove lines 12-14 to remove the header, and Jekyll will then use that file as a template instead of the default template that comes with the theme.</p>

**Comment by Sangarang** [comment]
<p>Thank you for your help, but I realised that my index file had the extension ‘.md’ instead of ‘.html’. So I renamed it to ‘index.html’.</p>

**Comment by zybbigpy** [comment]
<p>Hi, I think we need to remove 13 - 15 lines in default.html. Is it right?</p>

**Comment by claytonmcnamara** [comment]
<aside class="quote no-group" data-username="nickcannariato" data-post="2" data-topic="10323">
<div class="title">
<div class="quote-controls"></div>
 nickcannariato:</div>
<blockquote>
<p>you’ll need to copy this file into your repository inside a folder called _layouts:</p>
</blockquote>
</aside>
<p>this is not very clear… how do we create a folder? I can copy and paste the file, but creating the _layouts folder is a whole different problem :frowning:</p>

**Reply by eeholmes** [reply]
To create the _layouts folder and the `default.html` file in that

1. Navigate to your repo
2. Click Add File
3. When it asks for the name of file, type `_layouts/default.html`
4. Copy in the https://github.com/pages-themes/primer/blob/master/_layouts/default.html code.
5. Delete the lines
```
      {% if site.title and site.title != page.title %}
      <h1><a href="{{ "/" | absolute_url }}">{{ site.title }}</a></h1>
      {% endif %}
```
6. Commit the change (i.e. the added file)

The GitHub Actions bot will get to work and remake your page. You can watch its progress by clicking the Actions tab.

**Reply by rvcuenca** [reply]
Great! Finally found the answer I needed.
